node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodeAclResponse.java

```diff
@@ -49,6 +49,10 @@ private void toSlime(String hostname, Cursor object) {
         nodeRepository.getNodeAcls(node, aclsForChildren).forEach(nodeAcl -> toSlime(nodeAcl.trustedNetworks(),
                                                                                      nodeAcl.node(),
                                                                                      trustedNetworksArray));
+        Cursor trustedPortsArray = object.setArray("trustedPorts");
+        nodeRepository.getNodeAcls(node, aclsForChildren).forEach(nodeAcl -> toSlime(nodeAcl.trustedPorts(),
```

Comment:
Looks like we call `nodeRepository.getNodeAcls(node, aclsForChildren)` 3 times now, call it once and reuse result?

Reply:
One time it is